### ISSUE TITLE
Clearing Schematic Warnings, Rev 0.1, "Symbol 'GND' has been modified in library 'power'"

### ISSUE THREAD

**Issue by ForrestErickson** [PubInv/moonrat#98]
The schematic ERC reports 17 warnings. many related to symbols in the power library

![image](https://github.com/PubInv/moonrat/assets/5836181/c5375c32-c4f2-4493-9bc8-3aaf7e45fbff)

This is a question found on in the KiCad forum. See the traffic on the here:
https://forum.kicad.info/t/warning-symbol-gnd-has-been-modified-in-library-power/39116/

**Comment by ForrestErickson** [comment]
Lee had some back and forth with some Facebook KiCad users here:

![image](https://github.com/PubInv/moonrat/assets/5836181/4c204312-5f1e-49cc-9613-3bafa38105af)

**Comment by ForrestErickson** [comment]
Trying an update on the project pulled at 20231105 at 3:15 Am.

![image](https://github.com/PubInv/moonrat/assets/5836181/527b95e7-17dc-473c-9fc1-64e4f6e84960)

Change radio button for "Update symbols matching library identifier"
![image](https://github.com/PubInv/moonrat/assets/5836181/f389e694-9ce0-4f23-93ff-230631981393)


Pressed 'Update"
![image](https://github.com/PubInv/moonrat/assets/5836181/e3a48151-029c-4801-a5e2-37f1d695c421)

Seven locations were updated.
Reran the ERC and the warnings were reduce from 17 to 10.
![image](https://github.com/PubInv/moonrat/assets/5836181/7a984c91-531e-498d-a486-8320c4e3016e)

**Comment by ForrestErickson** [comment]
Now doing the same for the +5V symbol.

![image](https://github.com/PubInv/moonrat/assets/5836181/5d559c7c-9a30-49e3-a8cb-7ff2d51503f0)

![image](https://github.com/PubInv/moonrat/assets/5836181/5223527f-63a3-4b8d-878b-e1c7ca59cd2d)

Reduced the warnings again down to 8.
![image](https://github.com/PubInv/moonrat/assets/5836181/1dcb3178-e903-40f8-a8d8-849939c42899)

**Comment by ForrestErickson** [comment]
Same for PWR_FLG.

![image](https://github.com/PubInv/moonrat/assets/5836181/dd00fe38-a52a-4ba7-98cb-14c60a552cf4)

Same for power:VCC

![image](https://github.com/PubInv/moonrat/assets/5836181/19dd389e-31a7-45f6-9735-b2c2ecc42590)

Warnings further reduced

Deleted a floating "x" no connect flag.
![image](https://github.com/PubInv/moonrat/assets/5836181/8d896699-46c5-401e-a70f-98f7a26ddebf)

**Comment by ForrestErickson** [comment]
Regarding, 'TMP36xS' has been modified in library 'Sensor_Temperature'

![image](https://github.com/PubInv/moonrat/assets/5836181/c729c46f-9556-4867-95bb-f3e8d21c6140)

Lets update from library
![image](https://github.com/PubInv/moonrat/assets/5836181/15c426a5-199e-4c9c-acb3-0973a688e7eb)

**Comment by ForrestErickson** [comment]
Regarding
[lib_symbol_issues]: Symbol 'Conn_01x08_Male' not found in symbol library 'Connector'
    ; Severity: warning
    @(1200.00 mils, 4000.00 mils): Symbol J2 [Conn_01x08_Male]

![image](https://github.com/PubInv/moonrat/assets/5836181/cef3b170-8736-449a-98b6-ee27a261fc02)

For the OLED we used this symbol which is a KiCad connector. For the PCB foot print we then created a PCB footprint and put it in the connector properties.

Not sure the best way to clear this warning.
